tests/UsuariosTest.php

```diff
@@ -4,11 +4,30 @@
 
 class UsuariosTest extends \PHPUnit_Framework_TestCase
 {
-    public function testUsuarios()
+    protected $app;
+
+    public function setUp()
+    {
+	$api = new \ASC\Api();
```

Comment:
__[PHP_CodeSniffer]__ Spaces must be used to indent lines; tabs are not allowed ([link](https://sider.review/gh/repos/52858877/pulls/37#issue-3868105))

<sub>posted by Sider</sub>